pkg/sql/sem/tree/select.go

```diff
@@ -384,6 +384,8 @@ type AliasedTableExpr struct {
 	IndexFlags *IndexFlags
 	Ordinality bool
 	Lateral    bool
+	Only       bool
+	Descendant bool
```

Comment:
If you put `if node.Only { ctx.WriteString("ONLY") }` (and same for descendent), it will pretty print correctly and you'll be able to add this to `TestParse` instead of `TestParse2` :)

Reply:
We decided to not save the `Only` and `Descendant` fields into `AliasedTableExpr` so this is no longer relevant. The test remains in `TestParse2`